src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.csproj

```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <SignedPackageFile Include="analyzers/dotnet/cs/$(TargetFileName)" Certificate="$(AssemblySigningCertName)" />
+    <SignedPackageFile Include="$(TargetPath)" PackagePath="analyzers/dotnet/cs/$(TargetFileName)" Certificate="$(AssemblySigningCertName)" />
```

Comment:
Out of curiosity, would it be less effort to manage this if we used `<BuildOutputTargetFolder>` in https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.csproj to redirect the output and remove the nuspec?

Reply:
Yes. KoreBuild can automatically detect package layout when using nuspec generation. Explicit signing config is required when we have explicit .nuspec files.

Reply:
So should we try and avoid custom nuspecs if a project file would suffice (for instance in the case of the analyzer projects)?

Reply:
I'm leaning towards saying "keep using nuspec." Both of these are hard: making a .csproj pack a custom layout, or keeping signing config and nuspec layout aligned. We can't do much about the difficulties in getting nuspec generation to work, but we can make the sign tool smarter about custom nuspecs.